_topic_maps/_topic_map.yml

```diff
@@ -4666,6 +4666,13 @@ Topics:
     File: virt-configuring-higher-vm-workload-density
   - Name: Configuring certificate rotation
     File: virt-configuring-certificate-rotation
+- Name: Virtualization with IBM fusion access for SAN
```

Comment:
I would capitalize Fusion Access here and in the following Topics.Name fields

Reply:
Thanks. I agree. I had actually updated it in most places. Thanks for catching this one!